nodejs/week1/home/avg.js

```diff
@@ -0,0 +1,19 @@
+let sum = 0;
+
+if (process.argv.length === 2) {
+    console.log("Please enter a number");
+    return 0;
+}
+
+for (let i = 2; i < process.argv.length; i++) {
+    for (let j = 2; j < process.argv.length; j++) {
+        if (isNaN(process.argv[j])) {
+            console.log("Please enter a number");
+            return 0;
+        }
+    }
+    sum += parseInt(process.argv[i], 10);
+}
+
+const avg = sum / (process.argv.length - 2);
```

Comment:
It would make the code more readable if you created a variable that has the value `process.argv.length - 2`, and called something like `numberOfArguments`. That way you could also use it in your `for` loop conditions.